angular-hub/src/content/communities/angular-kenya.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Angular Kenya
```

Comment:
Please update the property from `name` to `title`

Reply:
Okay